NF.jsonld

```diff
@@ -18919,6 +18944,23 @@
             ],
             "sms:validationRules": []
         },
+        {
+            "@id": "bts:\"\"",
```

Comment:
Need to check this. Where is this random new key coming from?

Reply:
Figured out that if there is a space before the "" in valid values, jsonld will contain a key with id `"bts:\"\""`. Resolved with linting in https://github.com/nf-osi/nf-metadata-dictionary/pull/182/commits/05e5deffa3ba894c87c242ce16fbdafefb5fed58

Reply:
Interesting, thanks for catching that!